src/components/action-list/action-list.html

```diff
@@ -0,0 +1,160 @@
+<section class="docs-block">
+  <aside class="docs-block__info">
+    <h3 class="docs-block__header">Action list simple</h3>
+
+  </aside>
+  <div class="docs-block__content">
+    <div class="sg-actions-list sg-actions-list--space-between">
+      <div class="sg-actions-list__hole">
+        <button class="sg-button-secondary sg-button-secondary--small sg-button-secondary--inverse">
+          Answer
+        </button>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="docs-block">
+  <aside class="docs-block__info">
+    <h3 class="docs-block__header">Action list space between</h3>
+
+  </aside>
+  <div class="docs-block__content">
+    <div class="sg-actions-list sg-actions-list--no-wrap sg-actions-list--space-between-alt">
+
+      <div class="sg-actions-list__hole sg-actions-list__hole--no-shrink">
+
+        <div class="sg-label">
+
+          <div class="sg-label__icon">
+            <svg class="sg-icon sg-icon--x14 sg-icon--gray-secondary ">
+              <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-answer"></use>
+            </svg>
+          </div>
+
+          <div class="sg-text sg-text--obscure sg-text--emphasised sg-text--gray-secondary">
+            0/5
+          </div>
+
+        </div>
+
+      </div>
+
+      <div class="sg-actions-list__hole sg-actions-list__hole--no-shrink">
+
+        <div class="sg-label">
+
+          <div class="sg-label__icon">
+
+            <svg class="sg-icon sg-icon--x14 sg-icon--gray-secondary ">
+              <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-counter"></use>
+            </svg>
+          </div>
+
+          <div class="sg-text sg-text--gray-secondary sg-text--obscure sg-text--emphasised">
+            2d : 00h
+          </div>
+        </div>
+
+      </div>
+
+      <div class="sg-actions-list__hole sg-actions-list__hole--no-shrink">
+
+        <button class="sg-button-secondary sg-button-secondary--small sg-button-secondary--alt-inverse" title="Begin this challenge!">
+          Start
+        </button>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="docs-block">
+  <aside class="docs-block__info">
+    <h3 class="docs-block__header">Action list navigation</h3>
+  </aside>
+
+  <div class="docs-block__content">
+    <div class="docs-block__contrast-box">
+      <div class="sg-actions-list">
+        <div class="sg-actions-list__hole sg-actions-list__hole--no-spacing sg-actions-list__hole--container">
```

Comment:
`container` doesn't reveal much - name should probably suggest how it will behave IMO